doc/user-changes.md

```diff
@@ -0,0 +1,37 @@
+# User-facing changes log
```

Comment:
We will have to include this Markdown in the website doc page.

Reply:
I'll take a stab at it if you don't mind, so I can familiarize a bit more with the webpage generation.

Reply:
I'm reading through the sources for `alire.ada.dev` and if you can confirm that:

- [This line](https://github.com/alire-project/alire.ada.dev/blob/1be8728139d90f5c6d08481da7f7e762e0209585/update-gh-pages.sh#L49) is already including the file in the website generation.
- Because of [this configuration](https://github.com/alire-project/alire.ada.dev/blob/1be8728139d90f5c6d08481da7f7e762e0209585/_config.yml#L66), it may be better to make each entry in the log a third-level header so they don't clutter the TOC.

So no changes needed for the website itself?

Reply:
Here: https://github.com/alire-project/alire.ada.dev/blob/1be8728139d90f5c6d08481da7f7e762e0209585/docs/index.md

Each do file is included explicitly, which also controls how in what order they are in the table of content.

Reply:
I see, thanks.